fitparse/utils.py

```diff
@@ -56,3 +79,15 @@ def fileish_open(fileish, mode):
     else:
         # Python 3 - file contents
         return io.BytesIO(fileish)
+
+
+def is_iterable(obj):
+    """Check, if the obj is iterable but not string or bytes.
+    :rtype bool"""
+    if isinstance(obj, (str, bytes)):
+        return False
+    try:
```

Comment:
See https://stackoverflow.com/questions/1952464
`isinstance(obj, Iterable)` does not cover all use cases. However, for most of situations, isinstance is OK. Maybe should do it to speed up the code a bit.

Reply:
Huh, I actually had no idea that defining `__getitem__` on a class would cause it to become iterable. Seems like a legacy feature that was kept in for backwards compatibility after PEP234 was implemented. I did some tests in 2.7/3.6 and it seems that the basic types all implement `__iter__` now so as long as custom classes do the same it should be fine.